protobuf-definitions/make.bat

```diff
@@ -8,7 +8,7 @@
 
 SRC_DIR=proto/mlagents/envs/communicator_objects
 DST_DIR_C=../UnitySDK/Assets/ML-Agents/Scripts/CommunicatorObjects
-DST_DIR_P=../ml-agents
+DST_DIR_P=../ml-agents-envs
```

Comment:
Nee to change the windows make.bat as well

Reply:
Done(). Needs testing on Windows.